src/test/java/org/sonarsource/sonarlint/ls/mediumtests/AbstractLanguageServerMediumTests.java

```diff
@@ -442,18 +441,22 @@ private static DidChangeConfigurationParams changedConfiguration(@Nullable Strin
 
   protected static Map<String, Object> buildSonarLintSettingsSection(@Nullable String testFilePattern, @Nullable Boolean disableTelemetry, @Nullable Boolean showAnalyzerLogs,
     @Nullable Boolean showVerboseLogs, String... ruleConfigs) {
-    return buildSonarLintSettingsSection(testFilePattern, disableTelemetry, showAnalyzerLogs, showVerboseLogs, Collections.emptyMap(), ruleConfigs);
+    return buildSonarLintSettingsSection(testFilePattern, disableTelemetry, showAnalyzerLogs, showVerboseLogs, Collections.emptyMap(), null, ruleConfigs);
   }
 
   protected static Map<String, Object> buildSonarLintSettingsSection(@Nullable String testFilePattern, @Nullable Boolean disableTelemetry, @Nullable Boolean showAnalyzerLogs,
-    @Nullable Boolean showVerboseLogs, Map<String, String> analyzerProperties, String... ruleConfigs) {
+    @Nullable Boolean showVerboseLogs, Map<String, String> analyzerProperties, String pathToCompilationDatabase, String... ruleConfigs) {
     var values = new HashMap<String, Object>();
     if (testFilePattern != null) {
       values.put("testFilePattern", testFilePattern);
     }
     if (!analyzerProperties.isEmpty()) {
       values.put("analyzerProperties", analyzerProperties);
     }
+    if (StringUtils.isNotEmpty(pathToCompilationDatabase)) {
+      values.put("analyzerProperties", analyzerProperties);
```

Comment:
No, you should set configuration `pathToCompileCommands`, not an analyzer property...